### PR TITLE
[IOAP-399] Adds step to commit new app version number in CI process

### PR DESCRIPTION
**Pull request by sangalli**
* With this change, every change to a PR or to the main branch will trigger a commit in the same branch to update the app version number
* At this point the changes are all added to the workflow, but I'll probably move them to an action after working on [this other issue](https://vtex-dev.atlassian.net/browse/IOAP-400)
* The test of this change can be seen in this PR itself. The [5th commit](https://github.com/vtex-apps/starter.node-service/pull/13/commits/e34ec5c2daca90c9266f065150e1575b2e77b957) was performed by Github Actions Workflow

### CODE REVIEW

File: src/__tests__/middlewares/fetch-app-token-middleware.test.ts [hunk 1]
```diff
@@ -46,7 +46,6 @@ describe('Fetch App Token Middleware', () => {
       'x-vtex-api-appkey': thisAppKey,
       'x-vtex-api-apptoken': thisAppToken,
     })
-    expect(ctx.header?.['x-vtex-account']).toBe(accountName)
```
Comment (irgmedeiros) [inline src/__tests__/middlewares/fetch-app-token-middleware.test.ts:49]: Não precisa mais?

Comment (irgmedeiros) [reply src/__tests__/middlewares/fetch-app-token-middleware.test.ts:49]: Não acho que precisa de uma PR separada para uma mudança tão simples, assim tá bom

File: src/middlewares/app-routes.middleware.ts [hunk 2]
```diff
@@ -14,13 +14,17 @@ import koaCompose from 'koa-compose'
...
+import { validateExpectedHeaders } from './validate-expected-headers.middleware'
 
 export const createAppRoutesMiddleware = <T extends IOClients>(
   clientsConfig: ClientsConfig<T>
 ) => {
   const { implementation, options } = clientsConfig
 
   const middlewares = [
+    validateExpectedHeaders,
+    fetchAppTokenMiddleware,
```
Comment (irgmedeiros) [inline src/middlewares/app-routes.middleware.ts:27]: Não entendi porque `fetchAppTokenMiddleware` saiu do default middlewares e veio para cá. Pode me dar mais contexto?

Comment (sangalli) [reply src/middlewares/app-routes.middleware.ts:27]: Essa parte são os middlewares que são usados em rotas de app apenas. No caso das rotas de sistema (healthcheck no caso), imagino que não faça sentido chamar esse middleware. O que acha?

Comment (irgmedeiros) [reply src/middlewares/app-routes.middleware.ts:27]: Faz sentido

**Review by irgmedeiros — APPROVED** [review]
LGTM!

**Pull request merged**